apps/client/pages/_app.tsx

```diff
@@ -33,20 +33,34 @@ Sentry.init({
 })
 
 // Providers and components only relevant to a logged-in user
-const WithAuth = withAuthenticationRequired(function ({ children }: PropsWithChildren) {
-    return (
-        <ModalManager>
-            <UserAccountContextProvider>
-                <AccountContextProvider>
-                    {children}
+const WithAuth = function ({ children }: PropsWithChildren) {
+    const { data: session } = useSession()
+    const router = useRouter()
 
-                    {/* Add, edit, delete connections and manual accounts */}
-                    <AccountsManager />
-                </AccountContextProvider>
-            </UserAccountContextProvider>
-        </ModalManager>
-    )
-})
+    useEffect(() => {
+        if (!session) {
+            router.push('/login')
+        }
+    }, [session, router])
+
+    if (session) {
+        return (
+            <OnboardingGuard>
+                <ModalManager>
+                    <UserAccountContextProvider>
+                        <AccountContextProvider>
+                            {children}
+
+                            {/* Add, edit, delete connections and manual accounts */}
```

Comment:
I don't understand this comment, can you clarify what it's for? @tmyracle

Reply:
not sure, was there previously. I think it was to specify the purpose of the AccountsManager component (which will probably be refactored/removed)